app/controllers/api/v0/rounds/votes_controller.rb

```diff
@@ -8,19 +7,19 @@ def create
       render json: ErrorSerializer.new(ErrorMessage.new(exception.message, 404)).error_json, status: :not_found
     end
     if @round.open?
-      vote = Vote.find_by( user_id: params[:user_id],
-                           round_id: params[:round_id])
+      vote = Vote.find_by(user_id: params[:user_id],
+        round_id: params[:round_id])
       if vote
-        vote.update( lat: params[:lat], 
```

Comment:
This is one example where I liked the previous version more than the change. Having the extra indents serves a purpose in saying "This whole block serves one purpose, and you can skip over the rest if you know what the first line does". Without the extra indent, it looks just like another method at a glance.

Reply:
Currently updating to include greater indentation.